File: DotNet_Backend/SafarnamaApplication/SafarnamaApplication/DTOs/CustomerDTO.cs
namespace SafarnamaApplication.DTOs
{
    public class CustomerDTO
    {
        public int CustId { get; set; }
        public string? CustName { get; set; }
        public string? CustEmail { get; set; }
        public string? CustPhone { get; set; }
        public DateOnly? CustDob { get; set; }
        public string? CustGender { get; set; }
        public string? CustAddress { get; set; }
        public string? CustCity { get; set; }
        public string? CustState { get; set; }
        public string? CustPincode { get; set; }
        public string? CustCountry { get; set; }
        public DateTime? CustCreatedAt { get; set; }
    }
}

File: Java_Backend/E-Tour/src/main/java/dto/CustomerDTO.java
package dto;

import lombok.*;
import java.time.LocalDate;
import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class CustomerDTO {
    private Integer custId;
    private String custName;
    private String custEmail;
    private String custPhone;
    private LocalDate custDob;
    private Character custGender;
    private String custAddress;
    private String custCity;
    private String custState;
    private String custPincode;
    private String custCountry;
    private LocalDateTime custCreatedAt;
}
